src/scss/02_atoms/_buttons.scss

```diff
@@ -5,6 +5,7 @@ $buttonSizeConfig: (
     height: rem(40px),
```

Comment:
I bottoni lasciamoli come sono

Reply:
A volte avere questa classa in `flex` causa dei problemi di allineamenti a seconda del punto in cui si trova inserita.